### ISSUE TITLE
Navigable grid with virtual scrolling causes undesired behavior

### ISSUE THREAD

**Issue by svetq** [telerik/kendo-angular#2028]
## I'm submitting a...
* Bug report 

## Current behavior
The virtual scrolling example on the following article does not seem to function as expected, when using the material theme:
[https://www.telerik.com/kendo-angular-ui/components/grid/scroll-modes/virtual/#toc-getting-started](https://www.telerik.com/kendo-angular-ui/components/grid/scroll-modes/virtual/#toc-getting-started)

We need to set the [rowHeigh] accordingly, but even then if we use the down arrow key to move the grid, the page will be changed before we reach the last visible item on the current page.

## Minimal reproduction of the problem with instructions
Check the following example:
[https://stackblitz.com/edit/angular-8d5gaz-4gz7pz?file=app%2Fapp.component.ts](https://stackblitz.com/edit/angular-8d5gaz-4gz7pz?file=app%2Fapp.component.ts)

use the down arrow key to move down the grid

**Comment by szettel2** [comment]
I can observe the same issue with @progress/kendo-angular-grid@3.13.1 and the material theme if using Edge or FF. All seems to work ok in Chrome. 
In FF, i can just scroll one page (e.g. 100 items), in Edge moving the scrollbar knob to the end does not scroll to the last item of the grid.

**Comment by tsvetomir** [comment]
The [rowHeight] setting must match the actual row height. For the Material theme that's 49px, but depends on font size.

We recommend setting the row height using CSS and disabling word wrapping:
```css
    /*
     * Set a fixed row height of 36px (20px line height, 2 * 8px padding)
     *
     * [row height] = [line height] + [padding] + [border]
     *
     * Note: If using the Kendo UI Material theme, add 1px to the row height
     * to account for the bottom border width.
     */

    .k-grid tbody td {
        white-space: nowrap;
        line-height: 20px;
        padding: 8px 12px;
    }
```

The updated sample is available in the [development version](https://www.telerik.com/kendo-angular-ui-develop/components/grid/scroll-modes/virtual/) of the site.

**Comment by tsvetomir** [comment]
Original issue is fixed in the [latest development build](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/).

**Comment by danielkaradachki** [comment]
Fixed in 3.14.0:

https://stackblitz.com/edit/angular-8d5gaz-fkuaxg?file=app/app.component.ts
